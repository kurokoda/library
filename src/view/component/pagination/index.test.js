import React from "react";
import renderer from "react-test-renderer";

import Pagination from ".";

const props = {
  count: 0,
  countPerPage: 100,
  onPageChange: () => {},
};

it("renders correctly", () => {
  const tree = renderer.create(<Pagination {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
