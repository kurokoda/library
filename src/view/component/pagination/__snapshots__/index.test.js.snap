// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  aria-label="Page navigation example"
  className=""
>
  <ul
    className="pagination pagination-sm"
  >
    <li
      className="page-item disabled"
    >
      <button
        aria-label="First"
        className="page-link"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          «
        </span>
        <span
          className="sr-only"
        >
          First
        </span>
      </button>
    </li>
    <li
      className="page-item disabled"
    >
      <button
        aria-label="Previous"
        className="page-link"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          ‹
        </span>
        <span
          className="sr-only"
        >
          Previous
        </span>
      </button>
    </li>
    <li
      className="page-item"
    >
      page 
      1
       of 
      0
    </li>
    <li
      className="page-item"
    >
      <button
        aria-label="Next"
        className="page-link"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          ›
        </span>
        <span
          className="sr-only"
        >
          Next
        </span>
      </button>
    </li>
    <li
      className="page-item"
    >
      <button
        aria-label="Last"
        className="page-link"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
        >
          »
        </span>
        <span
          className="sr-only"
        >
          Last
        </span>
      </button>
    </li>
  </ul>
</nav>
`;
